bindings/wasm/bindings.js

```diff
@@ -17,6 +17,9 @@ Module.setup = function() {
   if (_ManifoldInitialized) return;
   _ManifoldInitialized = true;
 
+  // warmup tbb for emscripten, according to
+  // https://github.com/oneapi-src/oneTBB/blob/master/WASM_Support.md#limitations
+  Module.initTBB();
```

Comment:
Is this including changes from the emscripten TBB PR? Should they be separate?

Reply:
Yes I can remove them.